frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -65,6 +68,21 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
     );
   }
 
+  const handleStorageClass = (sc: string, provisioner: string) => {
+    setStorageClass(sc);
+
+    if (_.isEqual(provisioner, 'kubernetes.io/no-provisioner')) {
```

Comment:
```suggestion
    if (provisioner === 'kubernetes.io/no-provisioner') {
```

Reply:
is there any reason of using this over `_.isEqual`